File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings~/python/altunityrunner/commands/FindObjects/get_all_elements.py
from altunityrunner.commands.command_returning_alt_elements import CommandReturningAltElements
class GetAllElements(CommandReturningAltElements):
    def __init__(self, socket,request_separator,request_end,appium_driver,camera_name,enabled):
        super().__init__(socket,request_separator,request_end,appium_driver)
        self.camera_name=camera_name
        self.enabled=enabled
    
    def execute(self):
        if self.enabled==True:
            data = self.send_data(self.create_command('findObjects','//*', self.camera_name ,'true'))
        else:
            data = self.send_data(self.create_command('findObjects','//*', self.camera_name ,'false'))

        return self.get_alt_elements(data)

File: Assets/AltUnityTester/AltUnityDriver/Commands/FindObjects/GetAllElements.cs
public class GetAllElements : AltBaseCommand
{
    string cameraName;
    bool enabled;
    public GetAllElements(SocketSettings socketSettings, string cameraName, bool enabled) : base(socketSettings)
    {
        this.cameraName = cameraName;
        this.enabled = enabled;
    }
    public System.Collections.Generic.List<AltUnityObject> Execute()
    {
        Socket.Client.Send(toBytes(CreateCommand("findObjects", "//*", cameraName, enabled.ToString())));
        string data = Recvall();
        if (!data.Contains("error:")) return Newtonsoft.Json.JsonConvert.DeserializeObject<System.Collections.Generic.List<AltUnityObject>>(data);
        HandleErrors(data);
        return null;
    }
}